src/Uno.Extensions.Storage.UI/KeyValueStorage/KeyChainKeyValueStorage.cs

```diff
@@ -222,6 +216,17 @@ protected override async ValueTask InternalSetAsync<T>(string name, T value, Can
 		}
 	}
 
+	static internal void LogEntitlementsError()
+	{
+		if (typeof(KeyChainKeyValueStorage).Log().IsEnabled(LogLevel.Error))
```

Comment:
Please don't use this in Extensions. Logger should be available and is preferrable from using the Uno Log method that does internal Uno logging

Reply:
fixed